src/components/table/footer/cell/style.css

```diff
@@ -0,0 +1,26 @@
+:host {
+	display: table-cell;
+	line-height: 5rem;
+	white-space: nowrap;
+	padding: 0.3rem 0 0.3rem 1rem;
+	box-shadow: 0 1px 0 0 rgb(var(--smoothly-light-shade));
```

Comment:
consider changing the variable to `--smoothly-color-shade` or `--smoothly-color-tint` so that it is affected by the color attribute of the table.

Reply:
Ah, yeah. Copy pasted I think, but will do!